ratis-server/src/main/java/org/apache/ratis/server/impl/RaftServerImpl.java

```diff
@@ -182,6 +182,7 @@ public long getLastAppliedIndex() {
 
   private final TransferLeadership transferLeadership;
   private final SnapshotManagementRequestHandler snapshotRequestHandler;
+  private final StepDownLeader stepDownLeader;
```

Comment:
This should be moved to LeaderStateImpl since it is a leader-only request.

Reply:
down